CMakeLists.txt

```diff
@@ -170,50 +170,55 @@ if (NOT "${Boost_VERSION}" STRLESS "106100")
 endif()
 
 function(vexcl_add_executables targetname)
+    add_library(${targetname} INTERFACE)
```

Comment:
Does this need a `Common` alias, similar to line 199 below?

Reply:
Ah, probably not, since here its a leaf target, and there a library is defined.

Reply:
It could, but since executables don't have aliases (those are the actual executable names), I didn't add one here. I could if you think it's clearer.

Reply:
No, I think it's fine the way it is.